chain/chain/src/test_utils/kv_runtime.rs

```diff
@@ -996,17 +996,22 @@ impl RuntimeAdapter for KeyValueRuntime {
         Ok(self.tries.get_view_trie_for_shard(ShardUId::new(0, shard_id), state_root))
     }
 
+    fn get_shard_layout(&self, _epoch_id: &EpochId) -> Result<ShardLayout, Error> {
+        #[allow(deprecated)]
+        Ok(ShardLayout::v0(self.num_shards, 0))
+    }
+
```

Comment:
Would it work with the ShardLayout::multi_shard instead of the deprecated v0? the multi_shard is intended to be the latest shard layout whereas v0 is deprecated. If it doesn't work in the "good old" kv_runtime it's fine to keep v0.

Reply:
Sure, I can update this.  The value is ignored (for now at least) anyway so doesn't really matter.